src/main/java/frc/robot/subsystems/Tabs/TabDriveBase.java

```diff
@@ -22,19 +22,19 @@
     // DriveBaseSubsystem object
     private DriveBaseSubsystem mDriveBaseSubsystem;
 
-    //private AHRS mGyro;
```

Comment:
Revert this so the gyro is removed from this file. It messes with the subsystem disabling and really shouldn't be here.